codegenerator/cli/src/config_parsing/system_config.rs

```diff
@@ -969,10 +969,10 @@ impl Event {
                         EventType::LogData
                     } else {
                         match event_config.name.as_str() {
-                            "Mint" => EventType::Mint,
-                            "Burn" => EventType::Burn,
-                            "Transfer" => EventType::Transfer,
-                            "Call" => EventType::Call,
+                            MINT_EVENT_NAME => EventType::Mint,
+                            BURN_EVENT_NAME => EventType::Burn,
+                            TRANSFER_EVENT_NAME => EventType::Transfer,
+                            CALL_EVENT_NAME => EventType::Call,
```

Comment:
Isn't it possible for these to be aliased?

Reply:
Possible, this is for the case when `type` is not specified and we try to derive it from name

Reply:
If they are aliased, then the `type` field should be used and we won't enter the match branch